sdk/src/types/globals.ts

```diff
@@ -3,8 +3,11 @@ export type FetchInterface = (input: string | any, init?: RequestInit | any) =>
 
 export type UrlInterface = typeof URL;
 
+/** While the standard envisages 'string | URL', some implementations (e.g. React Native) only accept 'string' */
+type WebSocketUrl = string;
+
```

Comment:
No real need to type alias a `string` here.
```suggestion
```

Reply:
This has been separated so we can add a clean comment